packages/app/src/messages/components/messages/messages.component.tsx

```diff
@@ -316,7 +316,7 @@ const Messages: FunctionComponent<Props> = ({
       )
       if (thread) {
         openThreadDetails(thread)
-      } else {
+      } else if (tmpActiveThread === undefined && thread === undefined) {
```

Comment:
Shouldn't it be covered by some test?

Reply:
Can be 😉 Hold my beer 🍺 😅

Reply:
https://github.com/mudita/mudita-center/pull/747/commits/ecdc99ecee38aea2b7131f1321401ef7a15a7797